evacuees/src/API/EMBC.Registrants.API/ProfilesModule/ProfileManager.cs

```diff
@@ -75,6 +78,20 @@ public async Task<string> SaveProfile(Profile profile)
             if (!await profileRepository.DoesProfileExist(profile.Id))
             {
                 await profileRepository.Create(profile);
+
+                // get user by BCServicesCardId
+                Profile newProfile = await profileRepository.Read(profile.Id);
+
+                if (profile != null && profile.ContactDetails.Email != null)
+                {
+                    // Send email notification of new registrant record created
+                    EmailAddress registrantEmailAddress = new EmailAddress
+                    {
+                        Name = profile.PersonalDetails.FirstName + " " + profile.PersonalDetails.LastName,
+                        Address = profile.ContactDetails.Email
+                    };
+                    SendRegistrationNotificationEmail(registrantEmailAddress);
```

Comment:
Wouldn't this send the email notification for every update to the profile?

Reply:
No, it should only send the email notification when a profile is created.